src/index.js

```diff
@@ -1 +1,49 @@
-import './index.css';
\ No newline at end of file
+import { ListItem, createListItem } from './modules/ListItemMod.js';
+import './index.css';
+import svg from './asset/resource/icons.svg';
+
+const item1 = new ListItem('Item1', 1);
+const item2 = new ListItem('Item2', 2, true);
+const item3 = new ListItem('Item3', 3);
+const item4 = new ListItem('Item4', 4, true);
+const item5 = new ListItem('Item5', 5);
+
+const arrayItems = [item1, item2, item3, item4, item5];
```

Comment:
Kindly render the list items in the order of their indexes.

Consider the code snippet below:

```
const arrayItems = [item1, item2, item3, item4, item5].sort((a, b) => a.index - b.index);
```

Reply:
Thank you for the suggestion. Added throught the commit 6f2515b87debe71dc30400b6a855d314d09452cb